### PR TITLE
update explorer link , fixes #683

### LINKED ISSUE #683: update explorer link

### PR DESCRIPTION
**Pull request by NemTechCN**

### CODE REVIEW

File: src/components/transaction-details/TransactionDetailsTs.ts [hunk 1]
```diff
@@ -27,4 +31,13 @@ export class TransactionDetailsTs extends Vue {
...
         return this.transaction.isTxConfirmed ? 'confirmed' : 'unconfirmed'
     }
+
+    openExplorer(transactionHash) {
+        const {explorerHead} = this
+        if (explorerHead && transactionHash) {
+            window.open(explorerHead + transactionHash, '_blank')
+            return
+        }
+        this.$Notice.warning({title:Message.PLEASE_SET_CORRECT_EXPLORER_URL_FIRST})
```
[the remark this answers is not in the document]
Comment (NemTechCN) [reply src/components/transaction-details/TransactionDetailsTs.ts]: fixed

File: src/components/transaction-details/TransactionDetailsTs.ts [hunk 2]
```diff
@@ -27,4 +31,13 @@ export class TransactionDetailsTs extends Vue {
         if (!this.transaction.rawTx.signer) return null
         return this.transaction.isTxConfirmed ? 'confirmed' : 'unconfirmed'
     }
+
+    openExplorer(transactionHash) {
```
[the remark this answers is not in the document]
Comment (NemTechCN) [reply src/components/transaction-details/TransactionDetailsTs.ts]: fixed in building  folder

File: src/store/app/index.ts [hunk 3]
```diff
@@ -104,6 +105,9 @@ const mutations: MutationTree<AppInfo> = {
     REMOVE_TEMPORARY_INFO(state: AppInfo) {
         delete state.loadingOverlay.temporaryInfo
     },
+    SET_EXPLORER_HEAD(state: AppInfo, explorerHead: string) {
```
[the remark this answers is not in the document]
Comment (NemTechCN) [reply src/store/app/index.ts]: fixed

File: src/components/transaction-details/TransactionDetailsTs.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (evias) [reply src/components/transaction-details/TransactionDetailsTs.ts]: how is this fixed?

File: src/config/view/node.ts [hunk 4]
```diff
@@ -39,4 +40,9 @@ export const nodeListConfig: Endpoint[] = [
     },
 ]
 
-export const explorerUrlHead = 'http://explorer.mt.nemtech.network/transaction/'
+export const explorerLinkList = [
+    {
+        explorerBasePath: 'http://explorer.mt.nemtech.network/transaction/',
```
Comment (evias) [inline src/config/view/node.ts]: wrong base ... `explorer.nemtech.network`.

File: src/config/view/node.ts [hunk 5]
```diff
@@ -39,4 +40,9 @@ export const nodeListConfig: Endpoint[] = [
...
+        networkType: NetworkType.MIJIN_TEST
```
Comment (evias) [inline src/config/view/node.ts]: default should be TEST_NET

**Comment by domakun** [comment]
> please delete the unused key PLEASE_SET_CORRECT_EXPLORER_URL_FIRST

fixed

File: src/config/view/node.ts (hunk 5 above)
Comment (domakun) [reply src/config/view/node.ts]: fixed

File: src/config/view/node.ts (hunk 4 above)
Comment (domakun) [reply src/config/view/node.ts]: fixed

**Pull request merged**